src/chainparams.h

```diff
@@ -64,8 +64,9 @@ struct ChainTxData {
 
 /**
  * CChainParams defines various tweakable parameters of a given instance of the
- * Bitcoin system. There are three: the main network on which people trade goods
- * and services, the public test network which gets reset from time to time and
+ * Bitcoin system. There are four: the main network on which people trade goods
+ * and services, the public test network which gets reset from time to time,
+ * a signet test network with extra block requirements (see BIP325) and
  * a regression test mode which is intended for private networks only. It has
  * minimal difficulty to ensure that blocks can be found instantly.
```

Comment:
nit: It might be better to move the explanation of each network to where each network is defined

Reply:
Agree. Better not to enumerate things here which can be documented where they're defined.

Reply:
Not sure here. I think there can be value in listing short descriptions in one place to have an overview. A more detailed description would belong with the individual chains, sure.

Reply:
Ah oops I didn't see this comment! I suppose the issue with listing them here is that people might forget to update it, which is exactly what happened